src/Fixer/Casing/LowercaseConstantsFixer.php

```diff
@@ -24,8 +27,25 @@
  *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-final class LowercaseConstantsFixer extends AbstractFixer
+final class LowercaseConstantsFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    /**
+     * Hold the function that will be used to convert the constants.
+     *
+     * @var callable
+     */
+    private $fixFunction;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        $this->fixFunction = 'strto'.strtolower($this->configuration['syntax']);
```

Comment:
Looks like magic to me - concatenating 2 strings to to have function name.

I'd use anonymous function instead.

Reply:
Going to fix this now.

Reply:
@kubawerlos Could you check again, I just pushed an update. Thx.

Reply:
Just for my personnal information, could you also explain why, in this specific case, using an anonymous function would be better ?

Reply:
we prefer lambda functions because;
- these are more explicit
- static code analyzers understand the code

Reply:
Perfect, thanks @SpacePossum !